src/cryptonote_protocol/cryptonote_protocol_handler.inl

```diff
@@ -298,7 +298,12 @@ namespace cryptonote
     int64_t diff = static_cast<int64_t>(hshd.current_height) - static_cast<int64_t>(m_core.get_current_blockchain_height());
     uint64_t abs_diff = std::abs(diff);
     uint64_t max_block_height = std::max(hshd.current_height,m_core.get_current_blockchain_height());
-    uint64_t last_block_v1 = m_core.get_testnet() ? 624633 : 1009826;
+    uint64_t last_block_v1 = HARDFORK_1_HEIGHT - 1;
+    if (m_core.get_testnet())
+    {
+      // TODO: relace (uint64_t)-1 with (H-1) where H is the v2 fork height on testnet. Or remove this line altogether if H >= HARDFORK_1_HEIGHT
+      last_block_v1 = std::min<uint64_t>(last_block_v1, (uint64_t)-1);
```

Comment:
That's a noop isn't it ?

Reply:
As in the comment, my intention is to replace `(uint64-t)-1` later if we decide to introduce a v2 testnet fork at a height lower than `HARDFORK_1_HEIGHT `.

Reply:
I've updated this part slightly so that it's a bit simpler.